public/pages/DetectorDetail/components/ConfirmModal/ConfirmModal.tsx

```diff
@@ -44,7 +44,7 @@ export const ConfirmModal = (props: ConfirmModalProps) => {
   return (
     <EuiModal onClose={props.onClose}>
       <EuiModalHeader>
-        <EuiModalHeaderTitle>{props.title}</EuiModalHeaderTitle>
+        <EuiModalHeaderTitle>{props.title}&nbsp;</EuiModalHeaderTitle>
```

Comment:
Why add `&nbsp;` here?

Reply:
Spacing issue where the last character (in this case a `?`) gets cut off by the close icon. This adds an extra space to always prevent this from happening.

![Screen Shot 2020-06-02 at 6 23 16 PM](https://user-images.githubusercontent.com/62119629/83585248-3705d900-a4fe-11ea-8aa5-9735ee2e850a.png)